assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -342,6 +342,9 @@ che.infra.kubernetes.service_account_name=NULL
 # This property deprecates `che.infra.kubernetes.cluster_role_name`.
 che.infra.kubernetes.workspace_sa_cluster_roles=NULL
 
+# Cluster roles to assign to user in his namespace
+che.infra.kubernetes.user_cluster_roles=NULL
```

Comment:
Did we have a similar thing in che-7 workspaces, if not why do we need it in devworkspaces?